src/sagas/search.js

```diff
@@ -41,15 +41,20 @@ function* autocomplete(action) {
     const uiLang           = yield select(settingsGetUILangSelector);
     const contentLanguages = yield select(settingsGetContentLanguagesSelector);
     const autocompleteId   = GenerateSearchId();
-    const request          = { q: query, ui_language: uiLang, content_languages: contentLanguages };
+    const request          = {
+      q: query,
+      ui_language: uiLang,
+      content_languages: contentLanguages,
+      autocomplete_id: autocompleteId,
```

Comment:
You call this feature "Don't sent search/autocomplte id...", but here you're deliberately sending it

Reply:
I need the ID to be sent to chronicles (history logger).
But if you notice we DON'T sent this to API autocomplete/search I removed that parameter from http call.
This will enable us to cache search results in NGINX

Reply:
Are you sure somebody will understand it a couple of years from now?
Maybe to send this parameter separately, not as a part of request?
Or some other way?
If not, then just a commend will do

Reply:
Good point. Fixed.